lib/fizzy/types.hpp

```diff
@@ -18,6 +18,9 @@ enum class ValType : uint8_t
     i64 = 0x7e,
 };
 
+// The byte meaning an empty type list.
+constexpr uint8_t EmptyType = 0x40;
```

Comment:
`BlockTypeEmpty` would be closer to the spec, if this is only for block types

Reply:
Also the constants in execute.cpp still use snake case https://github.com/wasmx/fizzy/blob/e7a14a73fbf6a8bba7f93b42fb0dd649adab1750/lib/fizzy/execute.cpp#L11

Reply:
But things related to wasm spec in types.hpp use CamelCase.

Reply:
True, I would rather rename those other ones